rasa/__main__.py

```diff
@@ -113,7 +114,10 @@ def main() -> None:
             set_log_and_warnings_filters()
             rasa.telemetry.initialize_telemetry()
             rasa.telemetry.initialize_error_reporting()
-            cmdline_arguments.func(cmdline_arguments)
+            if inspect.iscoroutinefunction(cmdline_arguments.func):
+                rasa.utils.common.run_in_loop(cmdline_arguments.func(cmdline_arguments))
+            else:
+                cmdline_arguments.func(cmdline_arguments)
```

Comment:
this is no longer needed, no?

Reply:
True! I removed the coroutine check and it calls directly `cmdline_arguments.func(cmdline_arguments)` as before.